index.js

```diff
@@ -27,7 +27,9 @@ function GoogleAssistant(authConfig, callback) {
   if (authConfig.oauth2Client) {
     // we are passing in a client that is already authed with Google
     assistant = new Assistant(authConfig.oauth2Client);
-    assistantReady();
+    process.nextTick( () => {
```

Comment:
Quick question: What does putting this into the next tick do? What problem were you seeing?

You can also replace your code with this:
`process.nextTick(assistantReady);`

Reply:
First of all I'm not 100% sure if this was the problem. 
The issue is, that you can only register for the ready event once the init has finished. 
But with a oauth2client setting the ready event is triggered directly. So the ready comes before I can register to it. (Not 100%ly sure).

However, it would also be okay to describe that the ready event is only relevant for the auth. With an own oauth client this event is not important.

Reply:
```javascript
process.nextTick(assistantReady);
```
good point, I personally had some reference issues in the past without an arrow function wrapper. However your proposal should work. Do you want me to test and change this?

Reply:
No need to test (I did already), but if you could update it, that'd be great. Everything else looks good!